### ISSUE TITLE
[FAQ] Additions

### ISSUE THREAD

**Issue by DJTB** [Kaniwani/kw-frontend#29]
Q: I unlocked the first two levels and have been practicing my vocabulary. So far so weird, it seems all my items keep appearing in apprentice even though I can recall I have gotten them right at least 3 times, shouldn't some of them be in guru Level? How many times do I have to get them right to be levelling those up just like in wanikani?

A: We consider the first answer to be equivalent to the lesson, so on KW it's four correct answers to reach Guru (whereas on WK, after you complete the lesson, your item is already at Apprentice 1 ). You still effectively have to answer four times on both platforms, but the first time on WK you don't really notice since it's bundled in the lesson. All the following timings/ranks are the same.

**Comment by DJTB** [comment]
Q: Why aren't these two items with same meanings combined.

A: We've combined answers when WK uses the *exact* same meaning for an item. However, if it's even slightly different then the synonyms won't apply automatically.

For example 父 & 父親 will be combined since their WK meaning is strictly "father".
But WK decided that お父さん should be "father, dad, papa".

We're not prepared to manually combine the 6000+ items, and some people would disagree with our combinations (perhaps you only want お父さん to be accepted for papa since that was how WK taught you). In certain situations the nuance is important to distinguish as well. So instead we added the "Add Answer Synonym" button so users can decide for themselves.

If you answer ちち and it's wrong, but want it to be accepted, then add it as an answer synonym, and next time it will be accepted as correct. There are a lot of these situations regarding words for girl/young girl/lady, forest/woods etc, so we leave it up to you.

If you want to clarify the meanings further - any meaning synonyms added on WK are added to the English question in the review. You could add "da" and "daddio" to 父 and the quiz will ask you: "father, da, daddio".

In the later levels with more specific/defined vocabulary (like ocean trench, unsealed item etc) the similarity becomes less of an issue.

**Comment by DJTB** [comment]
Q: Won't this mess with my Wanikani SRS?

A: The SRS is mostly a time management tool, not so much a memorization tool. It helps you to do the minimum amount of study and still (in theory) remember items. By doing the minimum on each item, you free up more time to memorize a larger volume of items. The purpose of reviews, then, is to assess your current level of memorization of each item to determine how much to adjust your exposure to the item and still achieve memorization.

The problem with minimal memorization is that you won't reach fluency. Granted, that's not the SRS's intent. It's supposed to cram your head full if information, so when you encounter it in your main practice (reading, listening, grammar study), you're less likely to have to look things up, and your real-world practice will flow more naturally.

Basically, more practice is almost always good. Just know that if the SRS is your primary means of study, studying an item shortly before it comes up for review will affect the SRS's assessment of your progress, and you may end up getting less exposure to an item than you really need.

**Comment by DJTB** [comment]
Q: Using the Microsoft IME fails to enter the sokuon symbol followed by tsu, such as みっつ or やっつ.
A: There is an outstanding issue with the Microsoft IME and the kana conversion script in use. There is no fix on the horizon, so we highly recommend using the [Google IME](https://www.google.co.jp/intl/ja/ime/) instead.
